api/scpca_portal/models/library.py

```diff
@@ -104,3 +107,53 @@ def get_formats_from_file_paths(cls, file_paths: List[Path]) -> List[str]:
         if any(path for path in file_paths if common.ANNDATA_EXT == path.suffix):
             formats.append(Library.FileFormats.ANN_DATA)
         return formats
+
+    @classmethod
+    def get_project_libraries_from_download_config(
+        cls, project, download_configuration: Dict
+    ):  # -> QuerySet[Self]:
+        if download_configuration not in common.GENERATED_PROJECT_DOWNLOAD_CONFIGURATIONS:
+            raise ValueError("Invalid download configuration passed. Unable to retrieve libraries.")
+
+        if download_configuration["metadata_only"]:
+            return project.libraries.all()
+
+        if download_configuration["includes_merged"]:
+            # If the download config requests merged and there is no merged file in the project,
+            # return an empty queryset
+            if (
+                download_configuration["format"] == "SINGLE_CELL_EXPERIMENT"
```

Comment:
```suggestion
                download_configuration["format"] == FileFormats.SINGLE_CELL_EXPERIMENT
```

Let's avoid hardcoding these. I think this is good for now but maybe we should consider having an instance method on `Project` that takes the format and returns the correct `includes_merged_*` attribute value.

Reply:
Yeah, that's a good catch. I'll make this update.